src/main/support.js

```diff
@@ -16,49 +12,30 @@ import currentVersion from "./version.evaluate.cjs";
  * @param {object} param0
  * @param {(string | object)[]=} param0.plugins Strings are resolved by `withPlugins`.
  * @param {string[]=} param0.pluginSearchDirs Added by `withPlugins`.
- * @param {boolean=} param0.showUnreleased
  * @param {boolean=} param0.showDeprecated
  * @param {boolean=} param0.showInternal
  * @return {{ languages: Array<any>, options: Array<NamedOptionInfo> }}
  */
 function getSupportInfo({
   plugins = [],
-  showUnreleased = false,
   showDeprecated = false,
   showInternal = false,
 } = {}) {
-  // pre-release version is smaller than the normal version in semver,
-  // we need to treat it as the normal one so as to test new features.
-  const version = currentVersion.split("-", 1)[0];
-
-  const languages = plugins
-    .flatMap((plugin) => plugin.languages || [])
-    .filter(filterSince);
+  const languages = plugins.flatMap((plugin) => plugin.languages || []);
 
   const options = arrayify(
     Object.assign({}, ...plugins.map(({ options }) => options), coreOptions),
     "name"
   )
-    .filter((option) => filterSince(option) && filterDeprecated(option))
+    .filter((option) => filterDeprecated(option))
     .sort((a, b) => (a.name === b.name ? 0 : a.name < b.name ? -1 : 1))
     .map(mapInternal)
     .map((option) => {
       option = { ...option };
 
-      if (Array.isArray(option.default)) {
-        option.default =
-          option.default.length === 1
-            ? option.default[0].value
-            : option.default
-                .filter(filterSince)
-                .sort((info1, info2) =>
-                  semverCompare(info2.since, info1.since)
-                )[0].value;
-      }
```

Comment:
Maybe I should keep this array thing, otherwise it will be a breaking change, plugin may use array as default value.

Reply:
Deprecated array, introduced function for non-primitive values, it will be safer (to avoid mutate).

Reply:
Shouldn't these things be JSON-serializable?

Reply:
Make sense, so I'll just put `[{value: []}]` back.

Reply:
Wait, we get the support info from `getSupportInfo` function, so it will return `[]`, then it's serializable.

Reply:
The option self is not serializable, there are functions for `exception`.

Reply:
Let's add a new `defaultValue` field?

Reply:
How will it be different from `default`?

Reply:
When they are array, `defaultValue` use the value directly, `default` use `default[0].value`, so we don't need ask plugin to update.

Reply:
So I can use `defaultValue: []` instead of `default: [{value: []}]`.

Reply:
Forget it, switched back to  `default: [{value: []}]`